applications: Matter Bridge: extend CLI with data provider setters (#13282)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

Matter Bridge
-------------

* Added:

  * Support for groupcast communication to the On/Off Light device implementation.

>   * Support for controlling the OnOff Light simulated data provider with CLI command.

Reviewer: ```suggestion
  * Support for controlling the OnOff Light simulated data provider by using shell commands.
```
Changed to use the same wording as is used in the documentation.
